ruby/vampires.rb

```diff
@@ -1,8 +1,48 @@
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+counter = 0
+while counter < employees do
 puts "What is your name?"
-name = gets.chomp
+user = gets.chomp.downcase
 puts "How old are you?"
-age = gets.chomp
+age = gets.chomp.to_i
+puts "What year were you born?"
+year = gets.chomp.to_i
 puts "Our company cafeteria serves garlic bread. Should we order some for you? (Y/N)"
-bread = gets.chomp
+bread = gets.chomp.downcase
 puts "Would you like to enroll in the company's health insurance? (Y/N)"
-insurance = gets.chomp
+insurance = gets.chomp.downcase
+
+actual_age = 2017 - year
+
+is_vampire = nil
+if (age == actual_age) && (bread == "y" || insurance == "y")
+  is_vampire = "Probably not a vampire."
+end
+if (age != actual_age) && (bread == "n" || insurance == "n")
+  is_vampire = "Probably a vampire."
+end
+if (age != actual_age) && (bread == "n" && insurance == "n")
+  is_vampire = "Almost certainly a vampire."
+end
+if (user == "drake cula" || user == "tu fang")

```

Comment:
Consider the optimum condition when running your code.  What can we do to run only necessary processes?  We know that if the employee has Drac or Fang in their name then they are definitely a vampire.  If that is the case, do we need to run any of the other checks?

Consider:

```
if name_matches_vampire_params
  "Definitely a vampire"
else
  if #other checks
    "Something"
  elsif #more conditions...

  else
  end
end
```